### DISCUSSION TITLE
Is there a way to remove the space at the lower bar?

### DISCUSSION THREAD

**Discussion by morenohernan** [zellij-org/zellij#2723] [Q&A]
Hi, 

I want to remove the space shown in here ( see attached) . It is the space below the Tip label...
<img width="1672" alt="image" src="https://github.com/zellij-org/zellij/assets/700619/be275c86-0271-430e-acaf-e5a3e7547c32">

Kindly let me know if it is possible. 
Thanks in advance,

**Comment by roland-5** [comment]
What terminal do you use? I use Alacritty and doesn't have anything under tip bar.

**Reply by morenohernan** [reply]
I use iterm2. I've found a solution though, by using custom layout based in the examples.

**Reply by roland-5** [reply]
That's good. My guess is, that space under tip bar is created from wrong scaled font size.
